### DISCUSSION TITLE
v2.0.1

### DISCUSSION THREAD

**Discussion by Zingzy** [Zingzy/fluent-2048#2] [Announcements]
# What’s New in This Version (Full changelog) 🎉

- **[NEW]!** You can now change the grid size from 4x4 to 5x5 and vice versa 📏
- **[NEW]!** Add 5 new color schemes 🎨

# Notes
- Always keep the **configs** folder (zipped with the `exe` file) in the **Same folder as the** `exe` **file** ***No matter which version you install***
- In case you mess up your `config files`, you can always re-download them from GitHub
- You will have to manually restart the app if you changed Window Style to `Win7` and Backdrop style to `Mica`/ `Mica Alt` at the same time

# Visual Previews

- **`📏 Grid Sizes`**

<table>
  <tr>
    <td width="50%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/mica.png' /></td>
    <td width="50%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/5x5.png' /></td>
  </tr>
  <tr>
    <td width='50%' align='center'>4x4</td>
    <td width='50%' align='center'>5x5</td>
  </tr>
</table>

- **`🌈 Color Schemes`**

<table>
  <tr>
    <td width="33%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/midnightSerenity.png' /></td>
    <td width="33%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/mysticForest.png' /></td>
    <td width="33%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/royalPalace.png' /></td>
  </tr>
  <tr>
    <td width='33%' align='center'>Midnight Serenity Theme</td>
    <td width='33%' align='center'>Mystic Forest Theme</td>
    <td width='33%' align='center'>Royal Palace Theme</td>
  </tr>
</table>

<br>

<table>
  <tr>
    <td width="50%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/winterWonderland.png' /></td>
    <td width="50%"><image src='https://raw.githubusercontent.com/zingzy/fluent-2048/main/screenshots/translucent.png' /></td>
  </tr>
  <tr>
    <td width='50%' align='center'>Winter Wonderland Theme</td>
    <td width='50%' align='center'>Translucent Theme</td>
  </tr>
</table>

# Download options

- **Compact version**: This is a self-contained version of the app that does not require installation. You can run it from any folder or external drive. However, you will need to keep the `configs` folder with the `exe` file.
- **Faster version**: This is a lighter and faster version of the app, but this is **not bundled** in a single file. **The `_internal` folder is very important and must be kept in the same directory as the exe file.**

What shall you choose?
- We would recommend the **faster version** as its smaller in size and faster, but if you are afraid you might mess up some important files, you can download the compact version

# Known Issues
Changing the Window Style to `Win7` and backdrop Style to `Mica`/ `Mica Alt` will result in `Mica`/ `Mica Alt` effect not getting rendered
-Temporary fix : Restart the APP manually

<hr /><em>This discussion was created from the release <a href='https://github.com/Zingzy/fluent-2048/releases/tag/v2.0.1'>v2.0.1</a>.</em>

**Comment by vihaanmody1** [comment]
Source code please?

**Reply by Zingzy** [reply]
Hello @vihaanmody1 I will push the source code in the next few days

**Reply by vihaanmody1** [reply]
Thank you
